components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/exception/ExceptionCodes.java

```diff
@@ -26,14 +26,18 @@
  */
 public enum ExceptionCodes {
 
-    API_ALREADY_EXISTS(900300, "The API already exists.");
+    API_ALREADY_EXISTS(900300, "The API already exists.", 400),
```

Comment:
Its better to explain in a comment what these parameters are (unique error code, error message, corresponding http status code to be returned)